examples/mailbox/views/mailbox.js

```diff
@@ -7,11 +7,16 @@ const nav = require('../elements/nav')
 module.exports = function (state, prev, send) {
   return html`
     <main class="mw5 mw7-ns center cf">
+      <button onclick=${goHome}>Home</button>
       ${pathname(state, prev, send)}
       ${nav(state, prev, send)}
       <section class="fl mt4 w-80 db">
         ${emailList(state, prev, send)}
       </section>
     </main>
   `
+
+  function goHome () {
+    send('location:set', { pathname: '/' })
```

Comment:
This does require a bit of clarification. A quick demo shows that `{ href: '/' }` changes `window.location` and reroutes the way I expect it should. However, the documentation was updated to use `{ pathname: '/'}` which doesn't update the `window.location`, nor does it re-route correctly (though it does appear to change the state).